components/sysutils/mdata-client/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Joyent, Inc.
+#
+
+include ../../../make-rules/shared-macros.mk
```

Comment:
Adding `PREFERRED_BITS=64` before the include and changing common targets `s/32/64/` makes sure 64-bit binary is build and placed to the (usually 32-bit) area of `/usr/bin/`. E.g. see `components/archiver/gtar/` component.

Any reason to go with 32-bit binaries? We prefer 64-bit where possible, certainly for new stuff.

Reply:
At the moment, the software is only built and shipped 32-bit on any platform.  The `Makefile` doesn't currently offer a way to add different `CFLAGS` in a useful way.  I can try and fix that upstream, and then come back to improve the package here, but I'd like to get these binaries shipped as 32-bit first.